### DISCUSSION TITLE
School Suggestions

### DISCUSSION THREAD

**Discussion by kaizenics** [jeoooo/PROJECTDESTINY#20] [General]
AISAT Davao
https://aisat.edu.ph/
https://student.aisatportal.com/

Brokenshire College
http://www.brokenshire.edu.ph/

Christian Colleges of Southeast Asia
https://ccsa.edu.ph/

**Comment by jeoooo** [comment]
GitHub Issue created 

https://github.com/jeoooo/PROJECTDESTINY/issues/21
